services/src/projects/postgres_projectdb.rs

```diff
@@ -40,6 +42,96 @@ where
     pub fn new(conn_pool: Pool<PostgresConnectionManager<Tls>>) -> Self {
         Self { conn_pool }
     }
+
+    async fn list_plots(
```

Comment:
validate permission via `PostgresContext::check_user_project_permission`

Reply:
checked in list